### PR TITLE
Installing CNV 2.0

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://jira.coreos.com/browse/CNV-1591

* Need to double check all names/titles before publishing

~~[05/28/19] To do: revise content per BZ1707562~~

### CODE REVIEW

**Comment by AsherShoshan** [comment]
Looks fine.
What about Uninstall procedure/documentation?

( I think that the operatorgroup should come together with the subscription, and should be encapsulated in the subscription create/remove)

File: modules/cnv-preparing-to-install.adoc
Comment (nellyc) [inline modules/cnv-preparing-to-install.adoc]: i think we should directly point here to RH CDN, no?

Comment (AsherShoshan) [reply modules/cnv-preparing-to-install.adoc]: But is it only D/S?
If yes for U/S, then also all images inside CSV (inside catalogsource) should also changed to U/S

**Comment by ousleyp** [comment]
@AsherShoshan 

> Looks fine.
> What about Uninstall procedure/documentation?

Here's a link to the PR with the uninstall assembly: https://github.com/openshift/openshift-docs/pull/15053

Latest preview build: http://file.bos.redhat.com/pousley/053119/cnv-2.0-remove/cnv/cnv_install/uninstalling-container-native-virtualization.html

Let me know how it looks if you get a chance! Thanks :)

File: modules/cnv-preparing-to-install.adoc
Comment (ousleyp) [reply modules/cnv-preparing-to-install.adoc]: > i think we should directly point here to RH CDN, no?

Fabian said: "We can and thus should replace the PUBLIC_REGISTRY part, or rather the complete `image:` part once we know it.

Ryan Hallisey and Simone Tiraboschi can provide this once the errata goes into REL_PREP"

Anything I should change before then, @nellyc ?

**Comment by ousleyp** [comment]
@openshift/team-documentation @openshift/team-aligned-docs 

PTAL, thanks!!

File: modules/cnv-preparing-to-install.adoc
Comment (fabiand) [reply modules/cnv-preparing-to-install.adoc]: I actually wonder if we already know these two parts (registry and tag) today

File: cnv/cnv_install/installing-container-native-virtualization.adoc
Comment (bgaydosrh) [inline cnv/cnv_install/installing-container-native-virtualization.adoc]: Probably don't need 'then' but that's minor :-)

File: modules/cnv-deploying-cnv.adoc
Comment (bgaydosrh) [inline modules/cnv-deploying-cnv.adoc]: Is it 'under' or 'in'?

**Review by bgaydosrh — COMMENTED** [review]
Probably not much use except as my first GH peer review. :-)

**Review by rthallisey — APPROVED** [review]

**Review by alexxa — CHANGES_REQUESTED** [review]
Adding a 'request change'. We can't post docs with a pointer to hco-bundle-registry:v2.0.0-6
@fabiand

**Comment by fabiand** [comment]
@alexxa thus you want to have the correct image registry+name+tag before merging the pr?

**Comment by alexxa** [comment]
@fabiand @ousleyp is there a (release blocker) jira/bugzilla ticket to ensure that (installation) docs contain a correct registry+name+tag? If so, I'm ok this PR to be merged, and needed updates will be added later accordingly.

**Comment by fabiand** [comment]
No, we don't have it atm, but yes, let's create it, then we can merge this
PR (and move the corresponding bug to ON_QA) but have a reminder to fix the
registry.

File: modules/cnv-deploying-cnv.adoc
Comment (ousleyp) [reply modules/cnv-deploying-cnv.adoc]: It's physically under the page title "Provided APIs" so I *think* it should be "under." You don't click a button or anything to get there once you're on the Installed Operators page.

File: modules/cnv-subscribing-to-hco-catalog.adoc
Comment (kalexand-rh) [inline modules/cnv-subscribing-to-hco-catalog.adoc]: s/4.1/{product-version}

File: cnv/cnv_install/installing-container-native-virtualization.adoc
Comment (kalexand-rh) [inline cnv/cnv_install/installing-container-native-virtualization.adoc]: Do you have a concept module that provides details about the installation process? I think this assembly might benefit from one, even if it's not much more than "You need to install OCP before you install CNV. CNV runs as an Operator in the `<namespace>` namespace for \<reasons>\. Here's anything else you need to consider before you install."

File: modules/cnv-subscribing-to-hco-catalog.adoc
Comment (kalexand-rh) [inline modules/cnv-subscribing-to-hco-catalog.adoc]: The reason why I think you need a short concept is that I got to this point and was like "why?"

maybe s/You can/Before you install {ProductName}.

File: modules/cnv-deploying-cnv.adoc
Comment (kalexand-rh) [inline modules/cnv-deploying-cnv.adoc]: I'd make all of your prereqs parallel. Sometimes they're "you" statements, and other times they just state the existence of a state. Compare this one to the assembly prereqs.

File: modules/cnv-deploying-cnv.adoc
Comment (kalexand-rh) [reply modules/cnv-deploying-cnv.adoc]: Please check the IBM Style Guide's section about "Graphical user interface elements." It sounds like it might be what ISG considers a "pane," which would mean "in" wins.

File: modules/cnv-deploying-cnv.adoc
Comment (kalexand-rh) [inline modules/cnv-deploying-cnv.adoc]: I'd remove this step. In the assembly, you can create a Next steps section to list what you want users to do after they install.

You can amend the previous step to say something like, "After all the pods display the \*Running\* state, you can access {ProductName}."

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions and requests, but it largely LGTM.

File: modules/cnv-preparing-to-install.adoc
Comment (nellyc) [reply modules/cnv-preparing-to-install.adoc]: yes please change to RH prod CDN & just keep tag 2.0.0

File: _topic_map.yml
Comment (ousleyp) [reply _topic_map.yml]: Will do this right before we merge.

File: modules/cnv-preparing-to-install.adoc
Comment (ousleyp) [reply modules/cnv-preparing-to-install.adoc]: @nellyc I updated it in this commit: https://github.com/openshift/openshift-docs/pull/14858/commits/bc34188a31bc82ac19c189c528d1ab9a446d5e18

**Comment by ousleyp** [comment]
@kalexand-rh I think this can be merged now (if you agree). @alexxa from QE gave the green light, and @rthallisey approved it as well. Let me know if any issues remain. Thanks!

**Pull request merged**
